lisa/operating_system.py

```diff
@@ -198,19 +329,58 @@ def name_pattern(cls) -> Pattern[str]:
     def _initialize_package_installation(self) -> None:
         self._node.execute("apt-get update", sudo=True)
 
-    def _install_packages(self, packages: Union[List[str]]) -> None:
+    def _install_packages(
+        self, packages: Union[List[str]], signed: bool = True
+    ) -> None:
         command = (
             f"DEBIAN_FRONTEND=noninteractive "
             f"apt-get -y install {' '.join(packages)}"
         )
-        self._node.execute(command, sudo=True)
+        if not signed:
+            command = command.__add__(" --allow-unauthenticated")
```

Comment:
Never use those __add__s, there is syntactic sugar to wrap them all.

Reply:
Please look at the latest code. These are all taken care of now.

Reply:
Yeah, not a fan of those "fixing per review commits". You'll squash before merging, right?